### PR TITLE
feat: first? iteration of the piano tutorial

### PR DESCRIPTION
**Pull request by michalsek**

### CODE REVIEW

File: packages/audiodocs/docs/fundamentals/making-a-piano-keyboard.mdx [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+  }, []);
+}
+```
+
+## Playing the sounds
+
+Now it is finally time to play the sounds, but still nothing new here. We will use the `AudioBufferSourceNode` and simply play the buffers.
+
+```tsx
+export default function SimplePiano() {
```
Comment (michalsek) [inline packages/audiodocs/docs/fundamentals/making-a-piano-keyboard.mdx:118]: make it a live example

File: packages/audiodocs/docs/fundamentals/making-a-piano-keyboard.mdx [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+  source.start();
+
+  playingNotesRef.current[which] = source;
+};
+```
+
+And finally we can implement the `onKeyPressOut` function
+
+```tsx
+const onKeyPressOut = (which: KeyName) => {
```
Comment (michalsek) [inline packages/audiodocs/docs/fundamentals/making-a-piano-keyboard.mdx:180]: live example

File: packages/audiodocs/docs/fundamentals/making-a-piano-keyboard.mdx [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+
+  source.start(tNow);
+  playingNotesRef.current[which] = { source, envelope, startedAt: tNow };
+};
+```
+
+and the `onKeyPressOut` function:
+
+```tsx
+const onKeyPressOut = (which: KeyName) => {
```
Comment (michalsek) [inline packages/audiodocs/docs/fundamentals/making-a-piano-keyboard.mdx:279]: live example

**Review by maciejmakowski2003 — APPROVED** [review]
Amazing work!!!

**Pull request merged**